h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_fail.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import print_function
+import sys
+import traceback
+
+sys.path.insert(1, "../../")
+import h2o
+from tests import pyunit_utils
+import os
+from h2o.exceptions import H2OResponseError
+
+
+def test_parser_svmlight_column_skip_not_supported():
+    print("Test that functions calling fail if skipped_columns is passed with svm file.")
+    # generate a big frame with all datatypes and save it to svmlight
+    nrow = 10
+    ncol = 10
+    seed = 12345
+
+    f1 = h2o.create_frame(rows=nrow, cols=ncol, real_fraction=0.5, integer_fraction=0.5, missing_fraction=0.2,
+                          has_response=False, seed=seed)
+    f2 = h2o.create_frame(rows=nrow, cols=1, real_fraction=1, integer_fraction=0, missing_fraction=0,
+                          has_response=False, seed=seed)
+    f1.set_name(0, "target")
+    f1 = f2.cbind(f1)
+
+    tmpdir = os.path.normpath(os.path.join(os.path.dirname(os.path.realpath('__file__')), "..", "results"))
```

Comment:
it should probably use 

    results_dir = pyunit_utils.locate("results")

Reply:
The reason why is not used here is most probably to create results directory if not exists. 
`results_dir = pyunit_utils.locate("results")`
give error if results directory does not exists.

Reply:
does this happen on jenkins as well? I think that when ran via `run.py` the results dir is always present

Reply:
When the folder is present, everything goes well, if not, then it fails with file not found exception. I think, this workaround is because of locale run. I have no problem to replace it with suggested solution.

Reply:
I use results_dir = pyunit_utils.locate("results")